integrations/pooltogether-v4/wagmi.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from '@wagmi/cli'
+import { react } from '@wagmi/cli/plugins'
+
+import { YIELD_SOURCE_PRIZE_POOL_ABI } from './abis/yield-source-prize-pool-abi'
+export default defineConfig({
+  out: './pooltogether-v4-wagmi.ts',
+  contracts: [
+    {
+      name: 'poolTogetherPrizePool',
+      // @ts-ignore
+      abi: YIELD_SOURCE_PRIZE_POOL_ABI,
```

Comment:
Avoid using `@ts-ignore` as much as possible, if you do so it will break wagmi's type assertion when using the contract hooks. You should always export your ABI `as const` [more info here](https://wagmi.sh/core/typescript). After fixing the ABI imports you'll be able to remove the `@ts-ignore` on the other parts of the integration when using the contract hooks, this hugely improves the experience of devs using this integration down the road 😄